runtime-manager/modules/ROOT/pages/rtm-agent-app-validator.adoc

```diff
@@ -6,18 +6,29 @@ endif::[]
 
 include::reuse::partial$non-inclusive-banner.adoc[]
 
-Use the Application Validator Service to create your own custom validators and prevent your application from being deployed if certain conditions are not met.
+The Application Validator Service allows you to create your own custom validators and prevent your application from being deployed if certain conditions are not met.
 Depending on the Mule Runtime Agent configuration, the service invokes your custom validators to enforce their business rules, such as signing validation, naming conventions or moratorium dates.
 
 image:app-validator-chart.png[Application Validation Flowchart]
 
-The following example creates a custom validator that checks the value of an application property and stops the deployment if it does not meet the specified criteria.
-
 == Create an Application Validator
 
-=== Create a Maven project
+You can follow these steps to create a custom validator that checks the value of an application property and stops the deployment if it does not meet the specified criteria:
```

Comment:
```suggestion
Follow these steps to create a custom validator that checks the value of an application property and stops the deployment if it does not meet the specified criteria:
```

Reply:
It is not clear what "it" refers to. Does it refer to the validator or to a property?